molgenis-platform-integration-tests/src/test/resources/conf/molgenis.properties

```diff
@@ -0,0 +1,6 @@
+#Molgenis server properties used at dev time
+db_user=molgenis
+db_password=molgenis
+db_uri=jdbc\:postgresql\://localhost/molgenis_test?reWriteBatchedInserts=true&autosave=CONSERVATIVE
```

Comment:
```reWriteBatchedInserts=true&autosave=CONSERVATIVE``` repeated in multiple places. Append them in Java or can we figure out another smart way to do this?